pwa/src/templates/categoriesDetailTemplate/CategoriesDetailTemplate.tsx

```diff
@@ -54,7 +52,35 @@ export const CategoriesDetailTemplate: React.FC<CategoriesDetailTemplateProps> =
           <LeadParagraph className={styles.description}>{portfolio.longDescription}</LeadParagraph>
         </div>
       )}
-      {component && (
+
+      <div className={styles.resultsDisplaySwitchButtons}>
+        <Button
+          className={styles.buttonIcon}
+          variant={filters.catagoryDisplayLayout === "table" ? "primary-action" : "secondary-action"}
+          onClick={() => setFilters({ ...filters, catagoryDisplayLayout: "table" })}
+        >
+          <FontAwesomeIcon icon={faTable} />
+          {t("Table")}
+        </Button>
+        <Button
+          className={styles.buttonIcon}
+          variant={filters.catagoryDisplayLayout === "cards" ? "primary-action" : "secondary-action"}
+          onClick={() => setFilters({ ...filters, catagoryDisplayLayout: "cards" })}
+        >
+          <FontAwesomeIcon icon={faGripVertical} />
+          {t("Cards")}
+        </Button>
+        <Button
+          className={styles.buttonIcon}
+          variant={filters.catagoryDisplayLayout === "layer" ? "primary-action" : "secondary-action"}
+          onClick={() => setFilters({ ...filters, catagoryDisplayLayout: "layer" })}
+        >
+          <FontAwesomeIcon icon={faLayerGroup} />
+          {t("Layers")}
+        </Button>
+      </div>
+
+      {components && (
         <div className={styles.solutions}>
           <div className={styles.solutionsHeader}>
             <span className={maxItems && styles.maxNumber}>
```

Comment:
Instead of using `maxItems && ...`, I'd opt to use `componentCount >= 100 && ...`.